pyproject.toml

```diff
@@ -5,18 +5,18 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "pypformat"
-version = "0.8"
+version = "1.0.pre.4"
```

Comment:
Change the version to `1.0.0` and change the development status to `production/stable`, after the release has been properly tested on the test pypi repository.